docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  django:
+    container_name: iogt-dg01
```

Comment:
I would just use `name` instead of `container_name`. Docker Compose will add the project name and numbering all by itself.

Reply:
I think `name` is supported by the `network` and `volume` blocks only. Am i missing something here?

Reply:
No, I was mistaken. What I was actually suggesting was to leave `container_name` unspecified i.e. remove it. Docker Compose should automatically name containers with the following scheme <project name>_<service name>_<container number> e.g. iogt_django_1.